### PR TITLE
Added Validation Form for the Email

### PR DESCRIPTION
**Pull request by Meri-MG**
In this pull request was added validation form for the email.
The user should enter the email input in lowercase letters.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -83,3 +83,25 @@ openPopup.forEach((e) => {
...
+  } else {
+    errorMsg.innerHTML = '';
+    id.style.border = '2px solid black';
+  }
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  validation(email, 1, 'Email should be written in lowercase');
+});
```
Comment (DanteAlonsoHT) [inline index.js]: When I try to submit my message even if I have my email in lowercase, the message can not be submitted for Uppercase false detection, check the image below.

- [x] Fix your function to only allow lowercase as a requirement to submit an email to you.

image

File: index.html [hunk 2]
```diff
@@ -206,10 +206,12 @@ <h2 id="conttitle">CONTACT US</h2>
...
-                <form action="https://formspree.io/f/xvodlvop" method="POST">
-                                <input type="text" id="name" placeholder="Enter your full name" maxlength="30" required>
-                                <input type="email" id="email" placeholder="jondoe@mail.com" required>
-                                <textarea name="text" id="textarea" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                <form action="https://formspree.io/f/xvodlvop" method="POST" id="form">
+                                <input type="text" id="name" placeholder="Enter your full name" maxlength="30">
+                                <div class="error"></div>
+                                <input type="email" id="email" placeholder="jondoe@mail.com">
+                                <div class="error"></div>
+                                <textarea name="text" id="textarea" cols="30" rows="10" maxlength="500" placeholder="Enter text here"></textarea>
```
Comment (DanteAlonsoHT) [inline index.html]: - [x] Remember to don't remove the `required` HTML attribute in your `input` and  `textarea` fields in your contact form.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Meri-MG and @Lockless404, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
### _Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 3]
```diff
@@ -83,3 +83,23 @@ openPopup.forEach((e) => {
...
+    id.style.border = '2px solid red';
+  } else {
+    errorMsg.innerHTML = '';
+  }
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  validation(email, 1, 'Email should be written in lowercase');
+});
```
Comment (DanteAlonsoHT) [inline index.js:106]: Everything is working fine, but the only thing, is when I'm trying to submit the form, the form is not submitted, only fix that detail to get approval 😉 

- [x] Fix submit form issues (Form is not submitted when you put the right data in the form) Check if the formspree.io is working properly in your form, or check your JS code.


image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @Meri-MG, 

Great work on making the changes requested! 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
### _Check the comments under the review._ 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by carlos-ssh — APPROVED** [review]
Hi @Meri-MG, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**